bigtop-deploy/puppet/modules/kafka/templates/server.properties

```diff
@@ -25,7 +25,11 @@ broker.id=<%= @broker_id %>
 port=<%= @port %>
 
 # Hostname the broker will bind to. If not set, the server will bind to all interfaces
+<% if !@bind_addr.nil? && !@bind_addr.empty? %>
+host.name=<%= @bind_addr %>
```

Comment:
`host.name` is deprecated in favor of `listeners` in the latest release of Kakfa (0.10.0).  This is fine for now, as Bigtop is using 0.8.1, but it's something to keep in mind for the future.